### PR TITLE
Issue 16 finetuning with documents dataset

### PR DESCRIPTION
**Pull request by raquelroman96**
- Created new annotations for DOCIMANv1 and DOCIMANv2 and corresponding dataset classes.
- Created arbitraryCls class to infer images without groundtruth.
- CAT-Net ready to perform classification and segmentation:
  - Classification: tools/train_DCT_cls.py to train the DCT_cls network and infer_cls.py to perform inference.
  - Segmentation tools/train.py to train the DCT network and infer.py to perform inference.
- stages/experiment/run_experiment_mlflow.py to train/evaluate both models and show results in MLflow.
- New plots shown in local folder or MLflow:
  - For classification (inference and each validation step during training): Histogram, QF accuracy table, ROC curve.
  - For segmentation (inference and each validation step during training): QF mIoU table.

### CODE REVIEW

**Comment by raquelroman96** [comment]
## MobileNetv3 
New scrips added:
- CAT_DCT_MobileNetv3.yaml: configuration file for MobileNet. For classification experiment MODEL NAME: network_DCT_MobileNet_cls and for segmentation MODEL NAME: network_DCT_MobileNet_seg.
- lib/models/network_DCT_MobileNet_cls.py: MobileNetv3 model for classification.
- lib/models/network_DCT_MobileNet_seg.py: MobileNetv3 model for segmentation.

To train/infer any model of MobileNet arguments need to be adapted, ie. To perform inference with the classification model of MobileNetv3 we need to change the cfg='experiments/CAT_DCT_only_cls.yaml' from CAT-Net to cfg='CAT_DCT_MobileNetv3.yaml', as well as the weights route and the model initialization: 
  model = eval('models.' + config.MODEL.NAME + '.get_mobileNet_cls')(config)

Weights of best results obtained for MobileNetv3 classification: /media/BM/networks/CATNET/best_MobileNetv3_cls_epoch_6.pth.tar
